templates/contest/list.html

```diff
@@ -3,19 +3,7 @@
     <meta name="description" content="The {{ SITE_NAME }}'s contest list - past, present, and future.">
 {% endblock %}
 
-{% block media %}
-    <style>
-        .time-left {
-            text-align: left;
-            color: #777;
-            padding-top: 0.5em;
-        }
-
-        .content-description ul {
-            padding: 0 !important;
-        }
-    </style>
-{% endblock %}
+{% block media %}{% endblock %}
```

Comment:
We can probably delete this empty block.

Reply:
Done. I saw no change in the html source.